BasicGates/BasicGates.ipynb

```diff
@@ -264,7 +264,37 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Task 1.7. Bell state change - 1\n",
+    "### Task 1.7. Global Phase Change",
+    "\n",
+    "**Input:** \n",
+    "\n",
+    "1) Pauli operator (μ) represented as Pauli.\n",
+    "\n",
+    "2) Angle alpha, in radians, represented as Double.\n",
+    "\n",
+    "3) A qubit in state β|0⟩ + γ|1⟩.\n",
```

Comment:
In notebook version of the kata we usually write qubit states as formulas - they look neater that way

```suggestion
    "3) A qubit in state $|\\psi\\rangle = \\beta |0\\rangle + \\gamma |1\\rangle$.\n",
```

The same applies to the state in the goal.

Reply:
I was wondering about this! Thanks for the clarification.